app/(dashboard)/mailboxes/[mailbox_slug]/settings/team/teamSetting.tsx

```
@@ -30,6 +32,23 @@ const TeamSetting = ({ mailboxSlug }: TeamSettingProps) => {
     );
   });
 
+  const { mutate: removeTeamMember, isPending: isRemoving } = api.mailbox.members.delete.useMutation({
```

ESLint

[error] 'removeTeamMember' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)